app/master/atom.py

```diff
@@ -28,3 +28,12 @@ def __init__(
         self.actual_time = actual_time
         self.exit_code = exit_code
         self.state = state
+
+    def api_representation(self):
+        return {
+            'command_string': self.command_string,
+            'expected_time': self.expected_time,
+            'actual_time': self.actual_time,
+            'exit_code': self.exit_code,
+            'state': self.state,
+        }
```

Comment:
Nice. In subjob.py there is a `get_atoms()` method that we could probably get rid of in favor of this now.